schema/dereferenced-release-schema.json

```diff
@@ -1215,11 +1215,11 @@
         "budget": {
           "type": "object",
           "title": "Budget",
-          "description": "Details of the budget to be used in a future contracting process.",
+          "description": "Details of the budget that will fund the eventual contract(s)",
```

Comment:
Let's use "future" instead of "eventual" everywhere, for consistency and for the avoidance of ambiguity.